scripts/abi_to_rc/README.md

```diff
@@ -0,0 +1,14 @@
+# Purpose
+The `abi_to_rc.py` script processes a contract's .abi file in order to generate an overview Ricardian Contract and a Ricardian Contract for each action. The overview Ricardian Contract provides a description of the contract's purpose and also specifies the contract's action(s), input(s), and input type(s). The action Ricardian Contract provides a description of the action's purpose and also specifies the action's input(s), and input type(s).
+
+## How to run
+`python abi_to_rc.py ${{smart-contract.abi}} rc-overview-template.md rc-action-template.md`
```

Comment:
If you use the `argparse` module as mentioned below, you can write this as `python abi_to_rc.py -h`.  That way if the arguments change, there's no need to update this document, and no chance it goes stale.